####### The service to set the current robot battery level

### Service inputs

# The current robot robot battery level 
int16 charge_level

---

### Service output

# The current robot battery level 
int16 charge_level
